rust/datafusion/src/execution/physical_plan/string_expressions.rs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I created a new one to not over-populate `expressions.rs`. If we accept this, we should prob. move `length` here.